pkgs/by-name/wa/wavelog/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nix-update-script,
+  php,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "wavelog";
+  version = "1.9.1";
+
+  src = fetchFromGitHub {
+    owner = "wavelog";
+    repo = pname;
+    rev = version;
+    hash = "sha256-BYCRqb27QWOo74w3O6tfZGEDF3UInsgshsIm9uxOm+8=";
+  };
+
+  installPhase = ''
+    mkdir $out/
+    cp -R ./* $out
```

Comment:
is there any hidden file required?

like files that start with a dot

this command would ignore that

Reply:
btw this looks like a trivial case where `ln -s $src $out` would solve